Content.Client/Chat/ChatManager.cs

```diff
@@ -104,5 +128,84 @@ private void _onChatBoxTextSubmitted(ChatBox chatBox, string text)
                 }
             }
         }
+
+        private void _onFilterButtonToggled(ChatBox chatBox, Button.ButtonToggledEventArgs e)
+        {
+            // TODO make toggled ALL button flip all button states programatically + visually
+            switch (e.Button.Name)
+            {
+                case "Local":
+                    _Localstate = !_Localstate;
+                    if (_Localstate)
+                    {
+                        _filteredChannels = ChatChannel.Local;
```

Comment:
Don't you want `|=` for these? Using `=` will REPLACE all other filters.

Reply:
Didn't know that was an operator! I'll implement that instead.